tests/scripts/minikube.sh

```diff
@@ -122,6 +122,7 @@ function copy_images() {
 # configure minikube
 KUBE_VERSION=${KUBE_VERSION:-"v1.14.1"}
 MEMORY=${MEMORY:-"3000"}
+VM_DRIVER=${VM_DRIVER:-""}
```

Comment:
Line 128 was added recently and likely resolves your issue here without your updates in this file.